iothub/service/src/ManagedIdentity.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// The managed identity used to access the storage account for IoT hub import and export jobs.
+    /// TODO link from service team: For more information, see <see href=""/>
```

Comment:
Still TODO, or do you have a link for this?

Reply:
Still a TODO. This is their current doc, that still needs a review and by the url, seems like it is not the final link - https://review.docs.microsoft.com/en-us/azure/iot-hub/iot-hub-managed-identity?branch=pr-en-us-154985